html/rendering/replaced-elements/attributes-for-embedded-content-and-images/img-aspect-ratio.html

```diff
@@ -31,6 +31,10 @@
   test_computed_style_aspect_ratio("input", {type: "submit", width: width, height: height}, "auto");
 }
 
+function assert_cs(img, val) {
```

Comment:
I think the name can be better, the focus should be on aspectRatio, not (just) computed style as is now the case.